src/tv2_afvd_showstyle/migrations/index.ts

```diff
@@ -233,6 +235,10 @@ export const showStyleMigrations: MigrationStepShowStyle[] = [
 	 */
 	removeSourceLayer('1.7.5', 'AFVD', 'studio0_graphicsIdent_persistent'),
 
+	mapGFXTemplateToDesignTemplate('1.7.5', 'AFVD', 'GFXTemplates', 'GFXDesignTemplates'),
```

Comment:
I think it would be a good idea to bump this to '1.7.6' since it's a new round of migrations

Reply:
Bumped it up in [73a488c](https://github.com/tv2/sofie-blueprints-inews/pull/149/commits/73a488c30f009df70d38e047371d4b47fdf749ec)